src/tools/ScenarioMeasurement/Util/Util.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" />
     <PackageReference Include="System.Security.Principal.Windows" />
+    <PackageReference Include="MSBuild.StructuredLogger" VersionOverride="2.2.386" />
```

Comment:
Why do we need to override the version?

Reply:
The project is using central package version management, so version numbers can either be overridden or defined in Versions.props. I moved it to Versions.props.